src/System.Management.Automation/engine/parser/GlobalAssemblyCache.cs

```diff
@@ -1,4 +1,5 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This file is copied from Roslyn which is uses Apache, not MIT.

Reply:
I'll put the big banner version of the Apache copyright here